### ISSUE TITLE
org.gwtproject.safehtml:safehtml has dependency on gwt-user

### ISSUE THREAD

**Issue by bbende** [hal/elemento#54]
I'm fairly new to all this stuff so I apologize if this is more of a question than an issue...

I noticed the org.gwtproject.safehtml:safehtml  still has a transitive dependency on gwt-user:
```
[INFO] |  \- org.gwtproject.safehtml:safehtml:jar:1.0-SNAPSHOT:compile
[INFO] |     \- com.google.gwt:gwt-user:jar:2.8.2:compile
[INFO] |        +- com.google.jsinterop:jsinterop-annotations:jar:sources:1.0.2:compile
[INFO] |        +- javax.validation:validation-api:jar:sources:1.0.0.GA:compile
[INFO] |        +- javax.servlet:javax.servlet-api:jar:3.1.0:compile
[INFO] |        \- org.w3c.css:sac:jar:1.3:compile
```
Wasn't the purpose of using this library to specifically avoid depending on gwt-user? or does it not matter because it is not actually being used?

Also, it would be helpful in the Getting Started section to mention that anyone declaring a dependency on elemento-core will also need to declare the Vertispan snapshot repository, otherwise it won't be able to resolve the 1.0.0-SNAPSHOT version of safehtml. It is easy to figure this out, but would save people a little bit of time.

**Comment by vegegoku** [comment]
Safehtml has dependency on dom and dom is not fully ported yet, so once dom porting is complete this gwt-user dependency should be fixed

**Comment by bbende** [comment]
Appreciate the response, feel free to close this as it was just a question and not a real issue.

I'm now realizing that even when using elemental2/elemento, you still currently need gwt-user to get EntryPoint, but I guess the goal is to not rely on anything else in gwt-user so that eventually it could go away.

Is there any chance of publishing a 0.8.2 with the event dependency fix?

**Comment by hpehl** [comment]
Thanks for the hint about `EntryPoint`. I'll release a 0.8.2 shortly.

**Comment by hpehl** [comment]
Just released 0.8.2 including the event dependency fix
